src/All.php

```diff
@@ -0,0 +1,155 @@
+<?php namespace Leean\Acf;
+
+/**
+ * Class to provide helpers for getting a all ACF fields for an entity.
+ */
+class All
+{
+	/**
+	 * Is the ACF plugin active.
+	 *
+	 * @return bool
+	 */
+	public static function is_active() {
+		return function_exists( 'get_field_object' );
+	}
+
+	/**
+	 * Apply the default transforms if any exist for this field type.
+	 *
+	 * @param array $field The field.
+	 * @return mixed
+	 */
+	private static function apply_default_transform( $field ) {
+		$class = '\\' . __NAMESPACE__ . '\\Transforms\\' . str_replace( '_', '', ucwords( $field['type'], '_' ) );
+
+		if ( method_exists( $class, 'apply' ) ) {
+			return call_user_func( [ $class, 'apply' ], $field );
+		}
+
+		return $field['value'];
+	}
+
+	/**
+	 * Get all field values.
+	 *
+	 * @param int $target The target object.
+	 * @return array
+	 */
+	private static function get_fields( $target = 0 ) {
+		$data = [];
+
+		if ( self::is_active() ) {
+			$fields = get_field_objects( $target );
+
+			if ( $fields ) {
+				foreach ( $fields as $field_name => $field ) {
+					$apply_default_transforms =
+						apply_filters( 'ln_acf_apply_default_transforms', true, $target, $field );
+
+					$value = $apply_default_transforms ? self::apply_default_transform( $field ) : $field['value'];
+
+					$parent = get_post( $field['parent'] );
+
+					if ( $parent ) {
+						$data[ $parent->post_excerpt ][ $field_name ] =
+							apply_filters( 'ln_acf_field', $value, $target, $field );
+					} else {
+						$data[ $field_name ] =
+							apply_filters( 'ln_acf_field', $value, $target, $field );
+					}
+					
+				}
+			}
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get the field value for a post.
+	 *
+	 * @param int $post_id The target post's id. Or leave blank for he current post if in the loop.
+	 * @param bool $include_wp_fields Whether or not to include the default WP fields.
+	 * @return array
+	 */
+	public static function get_post_fields( $post_id = 0, $include_wp_fields = true ) {
+		if ( $include_wp_fields ) {
+			$wp_fields = [
+				'post_id' => $post_id,
+				'permalink' => get_permalink( $post_id ),
+				'title' => get_the_title( $post_id ),
+				'content' => apply_filters( 'the_content', get_post_field( 'post_content', $post_id ) ),
+			];
+
+			return array_merge( $wp_fields, self::get_fields( $post_id ) );
```

Comment:
Because wp_parse_args is for merging an array of args with a set of default values. In this case I'm just merging 2 arrays.